Auto update failing for wasmedge

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [wasmedge](https://github.com/termux/termux-packages/blob/9d3b62fdd5310e5db8c9e443c01cd97916d281d0/packages/wasmedge) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating wasmedge [Current version: 0.14.0]
INFO: package being updated to 0.14.1.
Downloading https://github.com/WasmEdge/WasmEdge/releases/download/0.14.1/WasmEdge-0.14.1-src.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0 3336k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 3336k  100 3336k    0     0  4582k      0 --:--:-- --:--:-- --:--:-- 27.3M

You are about to commit these changes:

--------------------
diff --git a/packages/wasmedge/build.sh b/packages/wasmedge/build.sh
index 22ee9891b..f8cbcc44b 100644
--- a/packages/wasmedge/build.sh
+++ b/packages/wasmedge/build.sh
@@ -3,11 +3,10 @@ TERMUX_PKG_DESCRIPTION="A lightweight, high-performance, and extensible WebAssem
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_LICENSE_FILE="LICENSE, LICENSE.spdx"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.14.0"
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION="0.14.1"
 # Use source tarball from release assets to get VERSION file for proper version number
 TERMUX_PKG_SRCURL=https://github.com/WasmEdge/WasmEdge/releases/download/${TERMUX_PKG_VERSION}/WasmEdge-${TERMUX_PKG_VERSION}-src.tar.gz
-TERMUX_PKG_SHA256=3fc518c172329d128ab41671b86e3de0544bcaacdec9c9b47bfc4ce8b421dfd5
+TERMUX_PKG_SHA256=e5a944975fb949ecda73d6fe80a86507deb2d0a221b2274338807b63758350b4
 TERMUX_PKG_DEPENDS="libc++, libspdlog"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--------------------

bump(main/wasmedge): 0.14.1


Not committing to Git!

INFO: Trying to build package.
INFO: Package wasmedge exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building wasmedge for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   106k      0 --:--:-- --:--:-- --:--:--  106k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1432      0 --:--:-- --:--:-- --:--:--  1431
100   833  100   833    0     0   1432      0 --:--:-- --:--:-- --:--:--  1431
gpg: Signature made Wed 18 Sep 2024 06:36:49 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  1 1617k    1 30629    0     0  30253      0  0:00:54  0:00:01  0:00:53 30265
 49 1617k   49  797k    0     0   451k      0  0:00:03  0:00:01  0:00:02  451k
100 1617k  100 1617k    0     0   912k      0  0:00:01  0:00:01 --:--:--  911k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24102      0 --:--:-- --:--:-- --:--:-- 24130
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1472      0 --:--:-- --:--:-- --:--:--  1471
100   833  100   833    0     0   1472      0 --:--:-- --:--:-- --:--:--  1471
gpg: Signature made Tue 17 Sep 2024 12:26:03 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  23766      0 --:--:-- --:--:-- --:--:-- 23811
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1309      0 --:--:-- --:--:-- --:--:--  1307
gpg: Signature made Tue 17 Sep 2024 09:58:05 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fmt@1:9.1.0 if necessary...
Found fmt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fmt/fmt_1:9.1.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  157k  100  157k    0     0  2459k      0 --:--:-- --:--:-- --:--:-- 2492k
extracting fmt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libspdlog@1.14.1 if necessary...
Found libspdlog in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libspdlog/libspdlog_1.14.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 16  182k   16 30615    0     0  35500      0  0:00:05 --:--:--  0:00:05 35475
100  182k  100  182k    0     0   178k      0  0:00:01  0:00:01 --:--:--  178k
extracting libspdlog to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/WasmEdge/WasmEdge/releases/download/0.14.1/WasmEdge-0.14.1-src.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 3336k  100 3336k    0     0  7527k      0 --:--:-- --:--:-- --:--:-- 7527k
Applying patch: cmake-Helper.cmake.patch
Applying patch: fix-for-arm.patch
Applying patch: lib-loader-aot_section.cpp.patch
Applying patch: lib-system-allocator.cpp.patch
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  120k  100  120k    0     0   298k      0 --:--:-- --:--:-- --:--:--  298k
Archive:  /home/builder/.termux-build/wasmedge/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 18.0.2
-- The C compiler identification is Clang 18.0.2
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for C++ include filesystem
-- Looking for C++ include filesystem - found
-- Performing Test CXX_FILESYSTEM_NO_LINK_NEEDED
-- Performing Test CXX_FILESYSTEM_NO_LINK_NEEDED - Success
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- Looking for mmap
-- Looking for mmap - found
-- Looking for C++ include pwd.h
-- Looking for C++ include pwd.h - found
-- Configuring done (2.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_FIND_ROOT_PATH_MODE_LIBRARY
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Generating done (0.0s)
-- Build files have been written to: /home/builder/.termux-build/wasmedge/build
[1/96] Building CXX object lib/common/CMakeFiles/wasmedgeCommon.dir/hexstr.cpp.o
In file included from /home/builder/.termux-build/wasmedge/src/lib/common/hexstr.cpp:4:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/hexstr.h:19:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/string:625:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/string_view:213:
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__fwd/string_view.h:22:41: warning: 'char_traits<fmt::detail::char8_type>' is deprecated: char_traits<T> for T not equal to char, wchar_t, char8_t, char16_t or char32_t is non-standard and is provided for a temporary period. It will be removed in LLVM 19, so please migrate off of it. [-Wdeprecated-declarations]
   22 | template <class _CharT, class _Traits = char_traits<_CharT> >
      |                                         ^
/data/data/com.termux/files/usr/include/fmt/core.h:377:26: note: in instantiation of default argument for 'basic_string_view<fmt::detail::char8_type>' required here
  377 | template <typename Char> using std_string_view = std::basic_string_view<Char>;
      |                          ^~~~~
/data/data/com.termux/files/usr/include/fmt/core.h:467:50: note: in instantiation of template type alias 'std_string_view' requested here
  467 |                                       S, detail::std_string_view<Char>>::value)>
      |                                                  ^
/data/data/com.termux/files/usr/include/fmt/format.h:713:57: note: in instantiation of template class 'fmt::basic_string_view<fmt::detail::char8_type>' requested here
  713 | inline auto compute_width(basic_string_view<char8_type> s) -> size_t {
      |                                                         ^
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__string/char_traits.h:81:8: note: 'char_traits<fmt::detail::char8_type>' has been explicitly marked deprecated here
   81 | struct _LIBCPP_DEPRECATED_(
      |        ^
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__config:916:53: note: expanded from macro '_LIBCPP_DEPRECATED_'
  916 | #      define _LIBCPP_DEPRECATED_(m) __attribute__((__deprecated__(m)))
      |                                                     ^
1 warning generated.
[2/96] Building CXX object lib/common/CMakeFiles/wasmedgeCommon.dir/spdlog.cpp.o
In file included from /home/builder/.termux-build/wasmedge/src/lib/common/spdlog.cpp:4:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/spdlog.h:16:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/filesystem.h:50:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/filesystem:539:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__filesystem/directory_entry.h:20:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__filesystem/filesystem_error.h:15:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__filesystem/path.h:27:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/string:625:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/string_view:213:
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__fwd/string_view.h:22:41: warning: 'char_traits<fmt::detail::char8_type>' is deprecated: char_traits<T> for T not equal to char, wchar_t, char8_t, char16_t or char32_t is non-standard and is provided for a temporary period. It will be removed in LLVM 19, so please migrate off of it. [-Wdeprecated-declarations]
   22 | template <class _CharT, class _Traits = char_traits<_CharT> >
      |                                         ^
/data/data/com.termux/files/usr/include/fmt/core.h:377:26: note: in instantiation of default argument for 'basic_string_view<fmt::detail::char8_type>' required here
  377 | template <typename Char> using std_string_view = std::basic_string_view<Char>;
      |                          ^~~~~
/data/data/com.termux/files/usr/include/fmt/core.h:467:50: note: in instantiation of template type alias 'std_string_view' requested here
  467 |                                       S, detail::std_string_view<Char>>::value)>
      |                                                  ^
/data/data/com.termux/files/usr/include/fmt/format.h:713:57: note: in instantiation of template class 'fmt::basic_string_view<fmt::detail::char8_type>' requested here
  713 | inline auto compute_width(basic_string_view<char8_type> s) -> size_t {
      |                                                         ^
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__string/char_traits.h:81:8: note: 'char_traits<fmt::detail::char8_type>' has been explicitly marked deprecated here
   81 | struct _LIBCPP_DEPRECATED_(
      |        ^
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__config:916:53: note: expanded from macro '_LIBCPP_DEPRECATED_'
  916 | #      define _LIBCPP_DEPRECATED_(m) __attribute__((__deprecated__(m)))
      |                                                     ^
1 warning generated.
[3/96] Building CXX object lib/system/CMakeFiles/wasmedgeSystem.dir/allocator.cpp.o
[4/96] Building CXX object lib/system/CMakeFiles/wasmedgeSystem.dir/mmap.cpp.o
[5/96] Building CXX object lib/common/CMakeFiles/wasmedgeCommon.dir/errinfo.cpp.o
FAILED: lib/common/CMakeFiles/wasmedgeCommon.dir/errinfo.cpp.o 
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27b-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DSPDLOG_SHARED_LIB -I/home/builder/.termux-build/wasmedge/build/include -I/home/builder/.termux-build/wasmedge/src/include -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wno-psabi -Wno-c++20-designator -Wno-c99-extensions -Wno-covered-switch-default -Wno-documentation-unknown-command -Wno-error=nested-anon-types -Wno-error=old-style-cast -Wno-error=shadow -Wno-error=unused-command-line-argument -Wno-error=unknown-warning-option -Wno-ctad-maybe-unsupported -Wno-gnu-anonymous-struct -Wno-keyword-macro -Wno-language-extension-token -Wno-newline-eof -Wno-shadow-field-in-constructor -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undefined-func-template -Wno-deprecated-literal-operator -Wno-switch-default -Wno-error=shadow-field -Wno-reserved-identifier -pthread -MD -MT lib/common/CMakeFiles/wasmedgeCommon.dir/errinfo.cpp.o -MF lib/common/CMakeFiles/wasmedgeCommon.dir/errinfo.cpp.o.d -o lib/common/CMakeFiles/wasmedgeCommon.dir/errinfo.cpp.o -c /home/builder/.termux-build/wasmedge/src/lib/common/errinfo.cpp
In file included from /home/builder/.termux-build/wasmedge/src/lib/common/errinfo.cpp:4:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/errinfo.h:16:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/enum_ast.hpp:22:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/spdlog.h:17:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/int128.h:469:
/data/data/com.termux/files/usr/include/fmt/format.h:3374:12: error: call to 'write' is ambiguous
 3374 |     return write<Char>(out, value);
      |            ^~~~~~~~~~~
/data/data/com.termux/files/usr/include/fmt/core.h:1639:12: note: in instantiation of function template specialization 'fmt::detail::default_arg_formatter<char>::operator()<unsigned __int128>' requested here
 1639 |     return vis(detail::convert_for_visit(arg.value_.uint128_value));
      |            ^
/data/data/com.termux/files/usr/include/fmt/core.h:3215:11: note: in instantiation of function template specialization 'fmt::detail::vformat_to<char>' requested here
 3215 |   detail::vformat_to(buf, fmt, args, {});
      |           ^
/data/data/com.termux/files/usr/include/fmt/core.h:3235:10: note: in instantiation of function template specialization 'fmt::vformat_to<std::back_insert_iterator<fmt::basic_memory_buffer<char>>, 0>' requested here
 3235 |   return vformat_to(out, fmt, fmt::make_format_args(args...));
      |          ^
/home/builder/.termux-build/wasmedge/src/include/common/types.h:664:10: note: in instantiation of function template specialization 'fmt::format_to<std::back_insert_iterator<fmt::basic_memory_buffer<char>>, const std::string_view &, const std::string_view &, 0>' requested here
  664 |     fmt::format_to(std::back_inserter(Buffer), "{} {}"sv,
      |          ^
/data/data/com.termux/files/usr/include/fmt/format.h:2218:20: note: candidate function [with Char = char, OutputIt = fmt::appender, T = unsigned __int128, $3 = 0]
 2218 | FMT_CONSTEXPR auto write(OutputIt out, T value) -> OutputIt {
      |                    ^
/data/data/com.termux/files/usr/include/fmt/format.h:3341:20: note: candidate function [with Char = char, OutputIt = fmt::appender, T = unsigned __int128, Context = fmt::basic_format_context<fmt::appender, char>]
 3341 | FMT_CONSTEXPR auto write(OutputIt out, const T& value) -> enable_if_t<
      |                    ^
1 error generated.
[6/96] Building CXX object lib/system/CMakeFiles/wasmedgeSystem.dir/path.cpp.o
[7/96] Building CXX object lib/system/CMakeFiles/wasmedgeSystem.dir/fault.cpp.o
[8/96] Building CXX object lib/plugin/wasi_logging/CMakeFiles/wasmedgePluginWasiLogging.dir/func.cpp.o
FAILED: lib/plugin/wasi_logging/CMakeFiles/wasmedgePluginWasiLogging.dir/func.cpp.o 
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27b-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DSPDLOG_SHARED_LIB -I/home/builder/.termux-build/wasmedge/build/include -I/home/builder/.termux-build/wasmedge/src/include -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wno-psabi -Wno-c++20-designator -Wno-c99-extensions -Wno-covered-switch-default -Wno-documentation-unknown-command -Wno-error=nested-anon-types -Wno-error=old-style-cast -Wno-error=shadow -Wno-error=unused-command-line-argument -Wno-error=unknown-warning-option -Wno-ctad-maybe-unsupported -Wno-gnu-anonymous-struct -Wno-keyword-macro -Wno-language-extension-token -Wno-newline-eof -Wno-shadow-field-in-constructor -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undefined-func-template -Wno-deprecated-literal-operator -Wno-switch-default -Wno-error=shadow-field -Wno-reserved-identifier -pthread -MD -MT lib/plugin/wasi_logging/CMakeFiles/wasmedgePluginWasiLogging.dir/func.cpp.o -MF lib/plugin/wasi_logging/CMakeFiles/wasmedgePluginWasiLogging.dir/func.cpp.o.d -o lib/plugin/wasi_logging/CMakeFiles/wasmedgePluginWasiLogging.dir/func.cpp.o -c /home/builder/.termux-build/wasmedge/src/lib/plugin/wasi_logging/func.cpp
In file included from /home/builder/.termux-build/wasmedge/src/lib/plugin/wasi_logging/func.cpp:7:
In file included from /home/builder/.termux-build/wasmedge/src/include/plugin/wasi_logging/func.h:9:
In file included from /home/builder/.termux-build/wasmedge/src/include/plugin/wasi_logging/base.h:9:
In file included from /home/builder/.termux-build/wasmedge/src/include/plugin/wasi_logging/env.h:9:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/hexstr.h:19:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/string:625:
In file included from /home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/string_view:213:
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__fwd/string_view.h:22:41: warning: 'char_traits<fmt::detail::char8_type>' is deprecated: char_traits<T> for T not equal to char, wchar_t, char8_t, char16_t or char32_t is non-standard and is provided for a temporary period. It will be removed in LLVM 19, so please migrate off of it. [-Wdeprecated-declarations]
   22 | template <class _CharT, class _Traits = char_traits<_CharT> >
      |                                         ^
/data/data/com.termux/files/usr/include/fmt/core.h:377:26: note: in instantiation of default argument for 'basic_string_view<fmt::detail::char8_type>' required here
  377 | template <typename Char> using std_string_view = std::basic_string_view<Char>;
      |                          ^~~~~
/data/data/com.termux/files/usr/include/fmt/core.h:467:50: note: in instantiation of template type alias 'std_string_view' requested here
  467 |                                       S, detail::std_string_view<Char>>::value)>
      |                                                  ^
/data/data/com.termux/files/usr/include/fmt/format.h:713:57: note: in instantiation of template class 'fmt::basic_string_view<fmt::detail::char8_type>' requested here
  713 | inline auto compute_width(basic_string_view<char8_type> s) -> size_t {
      |                                                         ^
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__string/char_traits.h:81:8: note: 'char_traits<fmt::detail::char8_type>' has been explicitly marked deprecated here
   81 | struct _LIBCPP_DEPRECATED_(
      |        ^
/home/builder/.termux-build/_cache/android-r27b-api-24-v1/sysroot/usr/include/c++/v1/__config:916:53: note: expanded from macro '_LIBCPP_DEPRECATED_'
  916 | #      define _LIBCPP_DEPRECATED_(m) __attribute__((__deprecated__(m)))
      |                                                     ^
In file included from /home/builder/.termux-build/wasmedge/src/lib/plugin/wasi_logging/func.cpp:7:
In file included from /home/builder/.termux-build/wasmedge/src/include/plugin/wasi_logging/func.h:9:
In file included from /home/builder/.termux-build/wasmedge/src/include/plugin/wasi_logging/base.h:9:
In file included from /home/builder/.termux-build/wasmedge/src/include/plugin/wasi_logging/env.h:10:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/spdlog.h:17:
In file included from /home/builder/.termux-build/wasmedge/src/include/common/int128.h:469:
/data/data/com.termux/files/usr/include/fmt/format.h:3374:12: error: call to 'write' is ambiguous
 3374 |     return write<Char>(out, value);
      |            ^~~~~~~~~~~
/data/data/com.termux/files/usr/include/fmt/core.h:1639:12: note: in instantiation of function template specialization 'fmt::detail::default_arg_formatter<char>::operator()<unsigned __int128>' requested here
 1639 |     return vis(detail::convert_for_visit(arg.value_.uint128_value));
      |            ^
/data/data/com.termux/files/usr/include/fmt/core.h:3215:11: note: in instantiation of function template specialization 'fmt::detail::vformat_to<char>' requested here
 3215 |   detail::vformat_to(buf, fmt, args, {});
      |           ^
/data/data/com.termux/files/usr/include/fmt/core.h:3235:10: note: in instantiation of function template specialization 'fmt::vformat_to<std::back_insert_iterator<fmt::basic_memory_buffer<char>>, 0>' requested here
 3235 |   return vformat_to(out, fmt, fmt::make_format_args(args...));
      |          ^
/home/builder/.termux-build/wasmedge/src/include/common/types.h:664:10: note: in instantiation of function template specialization 'fmt::format_to<std::back_insert_iterator<fmt::basic_memory_buffer<char>>, const std::string_view &, const std::string_view &, 0>' requested here
  664 |     fmt::format_to(std::back_inserter(Buffer), "{} {}"sv,
      |          ^
/data/data/com.termux/files/usr/include/fmt/format.h:2218:20: note: candidate function [with Char = char, OutputIt = fmt::appender, T = unsigned __int128, $3 = 0]
 2218 | FMT_CONSTEXPR auto write(OutputIt out, T value) -> OutputIt {
      |                    ^
/data/data/com.termux/files/usr/include/fmt/format.h:3341:20: note: candidate function [with Char = char, OutputIt = fmt::appender, T = unsigned __int128, Context = fmt::basic_format_context<fmt::appender, char>]
 3341 | FMT_CONSTEXPR auto write(OutputIt out, const T& value) -> enable_if_t<
      |                    ^
1 warning and 1 error generated.
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-09-18 18:43:58 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10927679238">10927679238</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/wasmedge/build.sh
+++ b/packages/wasmedge/build.sh
@@ -3,17 +3,18 @@ TERMUX_PKG_DESCRIPTION="A lightweight, high-performance, and extensible WebAssem
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_LICENSE_FILE="LICENSE, LICENSE.spdx"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.14.0"
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION="0.14.1"
 # Use source tarball from release assets to get VERSION file for proper version number
 TERMUX_PKG_SRCURL=https://github.com/WasmEdge/WasmEdge/releases/download/${TERMUX_PKG_VERSION}/WasmEdge-${TERMUX_PKG_VERSION}-src.tar.gz
-TERMUX_PKG_SHA256=3fc518c172329d128ab41671b86e3de0544bcaacdec9c9b47bfc4ce8b421dfd5
+TERMUX_PKG_SHA256=e5a944975fb949ecda73d6fe80a86507deb2d0a221b2274338807b63758350b4
 TERMUX_PKG_DEPENDS="libc++, libspdlog"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -DWASMEDGE_FORCE_DISABLE_LTO=ON
 -DWASMEDGE_USE_LLVM=OFF
 "
+# Until fmt 11.0.3 is released with https://github.com/fmtlib/fmt/issues/4140:
+TERMUX_PKG_BLACKLISTED_ARCHES="arm, i686"
 
 termux_step_pre_configure() {
 	case "${TERMUX_ARCH}" in

